src/BalconyBotanica/Core/DomainObjects/PlantData.cs

```diff
@@ -0,0 +1,13 @@
+namespace BalconyBotanica.Core.DomainObjects;
+public class PlantData
+{
+    public string Id { get; set; }
+    public string Common_name { get; set; }
+    public string[] Scientific_name { get; set; }
+    public string[] Other_name { get; set; }
+    public string Cycle { get; set; }
```

Comment:
I may be misunderstanding something with C# as I don't normally use it but I'm wondering why the enum types declared elsewhere in this directory cannot be used and everything is a string?

For example, can we have:

```
public class PlantData
{
  ...
  public Watering watering { get; set; }
  public Sunlight sunlight { get; set; }
}
```

Reply:
You are totally right. It was a lack of time to fully check my merge request. I've changed it! :)